### ISSUE TITLE
codebar for libraries

### ISSUE THREAD

**Issue by brunogirin** [codebar/chapter-organiser-meetings#26]
Due to budget cuts, my local council is closing libraries and my local one is moving to a community based model. The resident group who are bidding to run it had a meeting last night and they are very keen to offer computer coaching in the space. I mentioned codebar and they were very interested in the model.

My main question is: could a library work as a codebar chapter?

Key differences between a library (this one in particular) and a classic codebar host:
- the location is fixed,
- they have public computers on site which can open up the events to people who can't afford a laptop at home,
- mid to long term, they will want to extend the offering to children and senior citizens,
- this particular space is fully accessible which could extend the workshops to disabled people,
- as it's a community initiative, they may want to limit the workshops to people in the local community (I don't know this for sure but would be interested to discuss how that could work if it is the case).

In short I'd like to bring codebar to my local library and wonder how to do that in a way that is beneficial for both organisations.

**Comment by biggianteye** [comment]
Great idea to get more people into libraries!

Initially would it be feasible to set up this particular location as a host and see how that goes?

A couple of tricky things that come to mind that would need thinking about:
- food and drink: if we still want to do this, does the library have a suitable space for this? Also I suspect they won’t have the money for it, so a separate sponsor would probably be needed.
- catering to children and senior citizens: I think this is something coaches would need to be fully aware of upfront. Teaching children takes a different approach to teaching adults.
- catering to the local community: the current student and coach signup system is first come first served, so would require some manual management to get specific people in.

I appreciate that you’ve said some of those things are for the future, but just getting my thoughts out.

**Comment by brunogirin** [comment]
Thanks for the comments @biggianteye will take into account.

As discussed the monthly meeting on 30 Jan 2020, my local library is now fully going forward with a community system that will start in April and they will probably look at that sort of things in 6 to 9 months time. So the plan is to organise a meeting with them and @KimberleyCook when they are ready to discuss how that can work.
